### PR TITLE
FMD-233: Move Project Table Event Cols to JSON blob

### PR DESCRIPTION
**Pull request by MarcUsher**
### Change description
Migrates data in Project table columns that are fund-specific to a json_blob column called event_data_blob and subsequently drop the original columns

- [X] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [X] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

**Review by Joel-Pearce — APPROVED** [review]
LGTM

File: tests/db_tests/test_hybrid_properties.py [hunk 1]
```diff
@@ -54,12 +68,16 @@ def test_itl_regions_does_not_raise_uncaught_exception_with_invalid_postcode():
...
+        event_data_blob=json.dumps(
+            {
+                "primary_intervention_theme": "Theme 1",
+                "location_multiplicity": "MULTIPLE",
+                "locations": "Example Location",
+                "gis_provided": "Yes",
+                "lat_long": "12345",
+            }
+        ),
         postcodes=["ZZ1 2AB"],  # null postcodes
```
Comment (nsnape) [inline tests/db_tests/test_hybrid_properties.py:80]: not really part of this PR, but I noticed this comment is wrong. Could you update to say "invalid postcode"

File: db/migrations/versions/013_project_table_to_jsonb.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+"""empty message
+
+Revision ID: 013_project_table_to_jsonb
```
Comment (nsnape) [inline db/migrations/versions/013_project_table_to_jsonb.py:3]: just make sure you align with Joel on this version number, as I think he has an open branch with migration in too?

**Review by nsnape — CHANGES_REQUESTED** [review]

**Review by nsnape — APPROVED** [review]
